ioctl/cmd/node/nodedelegate.go

```diff
@@ -173,13 +173,16 @@ func delegates() error {
 		StartBlock:  int(epochData.Height),
 		TotalBlocks: int(response.TotalBlocks),
 	}
+
 	kickoutListRes, err := bc.GetKickoutList(epochNum)
 	if err != nil {
 		return output.NewError(0, "failed to get kickout list", err)
 	}
 	blacklist := &vote.Blacklist{}
-	if err := blacklist.Deserialize(kickoutListRes.Data); err != nil {
-		return output.NewError(output.SerializationError, "failed to deserialize kickout blacklist", err)
+	if kickoutListRes != nil {
+		if err := blacklist.Deserialize(kickoutListRes.Data); err != nil {
+			return output.NewError(output.SerializationError, "failed to deserialize kickout blacklist", err)
+		}
 	}
 	kickoutIntensityRate := float64(uint32(100)-blacklist.IntensityRate) / float64(100)
```

Comment:
Personally, I think puting deserialization into function `GetKickoutList` and letting it return Blocklist are better. Thus, it can return an empty Blocklist before Easter.

Reply:
to be consistent with other interface such as ```GetEpochMeta```